edocumentation-infra/src/main/java/org/dcsa/edocumentation/infra/enums/StringEnum.java

```diff
@@ -0,0 +1,3 @@
+package org.dcsa.edocumentation.infra.enums;
+
+public class StringEnum {}
```

Comment:
Should this have a `@UtilityClass` or a private constructor to ensure that cannot be instantiated? (mostly to prevent issues with a subclasses trying to do multiple things)